src/tooling/docs-assembler/assembler.yml

```diff
@@ -65,6 +65,8 @@ references:
   apm-aws-lambda:
   apm-k8s-attacher:
   beats:
+    current: "9.0"
+    next: main
```

Comment:
Can we have `next` as `9.1`? That would ensure we don't publish `main` again by accident and that once `v9.1.0` is released, there won't be any unreleased features in our docs.

Reply:
You didn't :)